lpcr_pos/static/src/xml/Screens/ProductScreen/ActionpadWidget.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+SPDX-FileCopyrightText: 2025 Coop IT Easy SC
+
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+<templates xml:space="preserve">
+    <t
+        t-name="pos_hide_refund_button.ActionpadWidgetHideRefundButton"
```

Comment:
`t-name` is not necessary, and moreover, if present, it should not include the module name (which here seems to come from a previous module) (unless in special cases like [here](https://github.com/OCA/pos/pull/1397/files#diff-5e48e266a0c215658c6ef251ff80377dd9de15ebc619ba66251db6096d2d44f1R14)).